Doc/library/contextlib.rst

```diff
@@ -433,24 +450,6 @@ statements to manage arbitrary resources that don't natively support the
 context management protocol.
 
 
-Simplifying support for single optional context managers
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I wonder if the section heading should be kept but the text changed to direct to nullcontext, so that people following links can see the new helper.

Also, if this is the only place that explained that ExitStack can work as a no-op, I think a line about that should be kept.

Reply:
Yes, it definitely should be kept.